src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/pages/Pages.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages
+
+sealed class Pages<T : ApplicationPage>(val page: Class<T>) {
```

Comment:
I would have liked to use an enum for this but you can not use type erasure like this in Kotlin enums hence the Sealed class.